models/books.js

```diff
@@ -0,0 +1,13 @@
+// entire book.js so far
+var mongoose = require('mongoose');
+
+var BookSchema = new mongoose.Schema({
+  title: String,
+  author: String,
+  image: String,
+  releaseDate: Date
+});
+
+var Book = mongoose.model('Book', BookSchema);
+
+module.exports = Book;
```

Comment:
Rename this to be `book.js` so it's singular instead of plural. It's only one book blueprint (and we like to stick to naming conventions.